src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,24 @@
-import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { filterSlice } from '../../features/filter';
+import { Status } from '../../types/Status';
+
+export const TodoFilter = () => {
+  const dispatch = useAppDispatch();
+  const { query, status } = useAppSelector(state => state.filter);
```

Comment:
Consider adding a type for `state` in the `useAppSelector` hook to ensure type safety and improve code readability.